packages/patterns-react/src/patterns/SimpleLongForm/README.md

```diff
@@ -0,0 +1,93 @@
+
+```
+SIMPLELONGFORM=true
+```
+
+> See
+> [feature-flags.md](https://github.com/carbon-design-system/ibm-dotcom-library/blob/master/packages/patterns-react/docs/feature-flags.md)
+> and
+> [.env.example](https://github.com/carbon-design-system/ibm-dotcom-library/blob/master/packages/patterns-react/.env.example)
+> for more information
+
+## Props
+
+| Name        | Description                                                                                                                 |
+| ----------- | --------------------------------------------------------------------------------------------------------------------------- |
+| `title`     | Simple long form title                                                                                                      |
+| `copy`      | Simple long form content                                                                                                    |
+| `variation` | Simple long form variation ( _standard_,_standard with simple link_, _standard with card link_, _standard with jump link_ ) |
+| `link`      | Simple long form link object with _href, text and target_ properties                                                        |
+
+## Variations
+
+| Name                   | Description                             |
+| ---------------------- | --------------------------------------- |
+| `standard`             | Default Simple long form without link   |
+| `standardWithJumpLink` | Default Simple long form with jump link |
+| `standardWithCardLink` | Default Simple long form with card link |
+
+## Stable selectors
+
+| Name                                 | Description |
+| ------------------------------------ | ----------- |
+| `bx--simplelongform`                 | Pattern     |
+| `bx--simplelongform--with-jump-link` | interactive |
+| `bx--simplelongform--with-card-link` | interactive |
+| `bx--simplelongform__container`      | interactive |
+| `bx--simplelongform__row`            | interactive |
+| `bx--simplelongform__col`            | interactive |
+| `bx--simplelongform__title`          | interactive |
+| `bx--simplelongform__content`        | interactive |
+| `bx--simplelongform__link__col`      | interactive |
+| `bx--simplelongform__divider__col`   | interactive |
+| `bx--simplelongform__divider`        | interactive |
+| `bx--simplelongform__link`           | component   |
+| `bx--simplelink`                     | interactive |
+| `bx--simplelink__text`               | interactive |
+| `bx--simplelink__icon`               | interactive |
+| `bx--jumplink`                       | component   |
+| `bx--jumplink__text`                 | interactive |
+| `bx--jumplink__icon`                 | interactive |
+| `bx--cardlink`                       | component   |
+| `bx--cardlink__card`                 | interactive |
+| `bx--cardlink__card__text`           | interactive |
+| `bx--cardlink__card__icon`           | interactive |
```

Comment:
stable selectors should be using the `dds` prefix. This is pulled from our settings, you can find examples in other components.

Reply:
@sannminwin the README still has the old prefix for stable selectors.

Reply:
yeah @jeffchew.  I am working on it. pushing commit one by one based on review comments.

Reply:
updated with [this commit](https://github.com/carbon-design-system/ibm-dotcom-library/pull/561/commits/717dd6b17118b19ee6179246a74f58cb34783d70)